charts/fluentd/templates/clusterrole.yaml

```diff
@@ -16,3 +16,10 @@ rules:
       - list
       - watch
 {{- end -}}
+{{- if .Values.podSecurityPolicy.enabled }}
+  - apiGroups: ['extensions']
+    resources: ['podsecuritypolicies']
+    verbs:     ['use']
+    resourceNames:
+    - {{ template "fluentd.fullname" . }}
```

Comment:
```suggestion
      - {{ include "fluentd.fullname" . }}
```
nit: Optional just for consistency sake, seems like lists are indented in this chart thus far and `include` is used over `template` even though both work in this case 🤷‍♂️